crates/sargon/src/system/drivers/secure_storage_driver/support/secure_storage_key.rs

```diff
@@ -19,13 +67,22 @@ impl SecureStorageKey {
                 SecureStorageKey::DeviceFactorSourceMnemonic {
                     factor_source_id,
                 } => format!("device_factor_source_{}", factor_source_id),
-                SecureStorageKey::ProfileSnapshot =>
+                SecureStorageKey::ProfileSnapshot { .. } =>
                     "profile_snapshot".to_owned(),
             }
         )
     }
 }
 
+impl SecureStorageKey {
+    pub fn load_profile_snapshot() -> Self {
+        let profile_id_noop = ProfileID::sample();
```

Comment:
So the ID is in fact never used, right? I think `noop` makes sense for a function, but this is a value (not fn type), so I'd prefer something like this:

```rs
pub fn load_profile_snapshot()-> Self {
    // This is in fact never used and later we are going to clean up this API
    let dummy_id = ProfileID(Uuid::from_bytes([0x00; 16])); 
    Self::ProfileSnapshot { profile_id: dummy_id }
}
```